tests/integration/cgroups.bats

```diff
@@ -12,45 +12,38 @@ function teardown() {
 function setup() {
     teardown
     setup_busybox
+    ROOTLESS_RES='"pids": { "limit": 100 },'
 }
 
-function check_cgroup_value() {
-    cgroup=$1
-    source=$2
-    expected=$3
-
-    current=$(cat $cgroup/$source)
-    echo  $cgroup/$source
-    echo "current" $current "!?" "$expected"
-    [ "$current" -eq "$expected" ]
-}
-
-@test "runc update --kernel-memory (initialized)" {
+@test "runc update --kernel-memory{,-tcp} (initialized)" {
     [[ "$ROOTLESS" -ne 0 ]] && requires rootless_cgroup
```

Comment:
`requires cgroups_v1` (explicitly)

Reply:
There is already `requires cgroups_kmem` below, and this is sufficient since v2 does not have kmem. 

Do you want to add an explicit `requires` just for the sake of readability?

Reply:
Either is fine to me